package.json

```diff
@@ -25,7 +25,8 @@
     "esbuild": "^0.14.53",
     "eslint-config-curvenote": "*",
     "prettier": "latest",
-    "turbo": "latest"
+    "turbo": "latest",
+    "ts-jest": "^28.0.7"
```

Comment:
We should bring jest here as well.

I tried to update every package to the same version of jest without things breaking.